src/chunk.c

```diff
@@ -837,6 +848,54 @@ chunk_find(Hyperspace *hs, Point *p)
 	return chunk;
 }
 
+static bool
+append_chunk_oid(ChunkScanCtx *scanctx, Chunk *chunk)
+{
+	if (chunk_is_complete(scanctx, chunk))
+	{
```

Comment:
This is a bit of a style choice, and not a big issue for a small function, but I tend to favor early exits, in this case:

```
if (!chunk_is_complete(scanctx, chunk))
    return false;
```

This makes code easier to read because you have less indentation and nesting and do not need go to the end of the function to know if the "negative" case means exit of executing some other code.